### DISCUSSION TITLE
Tesla AI day發布的Optimus和Boston dynamic Bot的區別

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#759] [General]
這個影片很詳細的說明今年Tesla發布的新機器人Optimus 與領先全球的波士頓動力機器人的區別。Tesla以電動車與其專利馬達以及控制系統為名，而波士頓動力機器人是以很靈活及超大扭力的跳舞機器人領先世界機器人，這兩家公司從不同的設計角度製作機器人，分享這個深入淺出的影片，講解Tesla機器人是Boston動力機器人的後腳跟，還是其實它所有的設計都是滿滿巧思。

https://youtu.be/pxGcGc9HXC4

**Comment by Zoe06060606** [comment]
這個影片講的很詳細 推推👍

**Comment by williamyen0623** [comment]
Tesla一直都是眾人視為很有潛力的公司，想必未來在機器人這塊領域也能發展得很好

**Comment by bonny123865** [comment]
我覺得影片當中特斯拉的機器人，在手部細節做了很大的改變!
然後Atlas跳舞好可愛，我重複看了好多次!!

**Comment by Yunhans** [comment]
太酷了吧，之後會不會便宜到每個人家裡都有一台機器人，我要拿來當我的管家

**Comment by evanchou1204** [comment]
Atlas跳舞真的很可愛，幾年前真的想不到好像科幻電影裡面的機器人會出現在真實世界，而且已經可以運動的那麼順暢，細思極恐。
